src/problem.cpp

```diff
@@ -277,6 +282,33 @@ activ::ProblemDescription Problem::AsActivation() const
     }
 }
 
+softmax::ProblemDescription Problem::AsSoftmax() const
+{
+    const auto& softmax_desc = boost::get<SoftmaxDescriptor>(operator_descriptor);
+
+    float alpha = softmax_desc.GetAlpha();
+    float beta  = softmax_desc.GetBeta();
+
+    softmax::ProblemDescription problem_description =
+        (GetDirection() == miopenProblemDirectionForward)
+            ? softmax::ProblemDescription(
+                  &alpha,
+                  &beta,
```

Comment:
[Performance] @Vsevolod1983 See https://github.com/ROCm/MIOpen/pull/2671#pullrequestreview-1937760018

Reply:
I removed alpha and beta from network config in softmax primitive. Is it enough for the given PR ?

Reply:
@Vsevolod1983 I would begin with removing alpha/beta from PD (and then fix build errors by forwarding alpha/beta via InvokeParams). However I think we need to discuss https://github.com/ROCm/MIOpen/pull/2671#pullrequestreview-1937760018 first.

/cc @CAHEK7 @DrizztDoUrden

Reply:
Should we do this kind of refactoring as a separate ticket / PR ?

Reply:
I restored alpha and beta in network config in this PR until wedecide what exactly we want to do next

Reply:
[Informative] @Vsevolod1983 @CAHEK7 it's interesting that "default alpha" optimization won't give us much, but "default beta" opt removes 1 global memory read, which may be substantial (./src/kernels/MIOpenSoftmax.cl)

Reply:
@atamazov right now `default alpha` gives much since it enables attention softmax solver, which is faster and does not support a/b in terms of normal softmax operation.

Both values are used in the `isApplicable` method and can't be just removed.